plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestingPostgreSql.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.postgresql;
+
+import org.intellij.lang.annotations.Language;
+
+import java.io.Closeable;
+
+public interface TestingPostgreSql
```

Comment:
The name is a bit confusing with `TestingPostgreSqlServer`, how about this?
```suggestion
public interface TestingServer
```